### ISSUE TITLE
Two disc albums don't play in order

### ISSUE THREAD

**Issue by neilmusgrove** [vially/googlemusic-xbmc#23]
I have a number of two disc albums with duplicate track numbers (as is standard when more than 1 disc), eg disc 1 track 1, disc 2 track 1. The official website, apps etc correctly play the second disc tracks after all the first disc tracks however this plugin plays them solely in track order hence the second disc tracks are mixed in with the first disc tracks,

Thanks

**Comment by foreverguest** [comment]
Hmm, you're playing by 'Album' and  the two discs have the same album name? Indeed in this case we are ordering only by track, should be easy to add order by disc if the info is available.

**Comment by neilmusgrove** [comment]
That's correct. Although previously they were named 'album name - disc 1'
etc it worked better on Play Music to have the same album name and specify
the disc field. It means you don't have duplicate albums in the list to
sort through and Play Music supports it really well. Thanks.

**Comment by foreverguest** [comment]
Please, could you test this version and check if it works for you:
https://www.box.com/s/wtcvhwcpm9mx0rbqpojf

**Comment by neilmusgrove** [comment]
Yes that fix has worked, second disc tracks now follow on from first disc tracks. Thanks for the quick fix. Issue 24 also appears to be fixed in this build.
